test/tektonbundles_test.go

```diff
@@ -54,6 +54,10 @@ var requireFeatureFlags = requireAnyGate(map[string]string{
 	"enable-api-fields":         "alpha",
 })
```

Comment:
can this variable (requireFeatureFlags) be removed now?

Reply:
That'll be part of #5514